poetry.lock

```diff
@@ -0,0 +1,3072 @@
+# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
+
+[[package]]
+name = "aiohappyeyeballs"
```

Comment:
👀

Reply:
Somehow it's a dependency, but the name is funny